### PR TITLE
Hit build script

### PR DESCRIPTION
**Pull request by derekstucki**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Allows hit to be compiled in windows.

##### What are the significant changes in functionality due to this change request?
Hit will always now be compiled during RAVEN make rather than MOOSE make for the RAVEN project.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: raven.mk [hunk 1]
```diff
@@ -9,13 +9,22 @@ SHELL := /bin/bash
 include $(RAVEN_DIR)/amsc.mk
 ###############################################################################
 
+<<<<<<< HEAD
```
Comment (alfoa) [inline raven.mk]: Conflict

**Comment by alfoa** [comment]
PS. can you add these modifications in the branch of  PR #518  (it is about devel) 
Thanks.
Andrea

File: raven.mk (hunk 1 above)
Comment (derekstucki) [reply raven.mk]: There were multiple conflicts in that file and apparently I forgot to check for remaining ones after fixing some.

**Comment by derekstucki** [comment]
This PR depends on at least commit 28027fd9b2 of MOOSE. Can it go into devel without the dataobject-rework branch?

**Comment by alfoa** [comment]
I think this PR should be for devel  (PR I pointed you to) and not for dataojbect rework.
Once that PR is accepted in devel, we can just rebase this branch without the need to create 2 PRs

**Comment by derekstucki** [comment]
Putting this fix in RAVEN has the same problem as putting it in MOOSE. Civet is not compatible with python's setuptools. Maybe it could be addressed in RAVEN's Civet recipe, but it won't work here without some really ugly hacks.

**Comment by alfoa** [comment]
@derekstucki 
As I mentioned in one of the comments above, this PR should be splitted in two PR
1) the updated moose submodule and hit compilation procedure should be added in the branch ```alfoa/testFix``` for which we have a PR ( #518 ) that fixes the regression test inconsistencies and targets the branch ```devel```
2) this PR should force and update the ```RavenUtils.py``` only.

Once the first PR ( #518) is in, we can rebase the dataobject rework branch and get the modifications and moose submodule update from devel.

Otherwise this PR will create tons of conflicts.

Thanks

**Comment by derekstucki** [comment]
I'll do that split as soon as we finalize the windows testing.

**Comment by derekstucki** [comment]
Only foreseeable problem left is updating moose on the windows test box.

**Comment by PaulTalbot-INL** [comment]
This was moved to a devel merge, and has been merged.

**Comment by alfoa** [comment]
We close this PR since the modifications here are in devel already.

**Pull request closed without merging**